### PR TITLE
new term entry math erf()

### LINKED ISSUE #5030: [Term Entry] Python math-module math.erf()
### Reason/inspiration (optional)

We would like a new term entry on the `math.erf()` method of the [`math-module`](https://www.codecademy.com/resources/docs/python/math-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-erf/math-erf.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by sicxz**
### Description

I created the file math.erf.md its contents are the term entry math.efr() for the Python math-module. Per instructions I used the term-entry-template.md, content standards, and markdown style guide. 

My motivation is to create a portfolio project for the codecademy course Learn Git and GitHub. 

### Issue Solved

Closes #5030 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
> [CLA assistant check](https://cla-assistant.io/Codecademy/docs?pullRequest=5048) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=5048) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=5048) it.

Hey @sicxz please sign the CLA so that the review can get started

File: content/python/concepts/math-module/terms/math-erf/math-erf.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Tags: 
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Statistics'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-erf/math-erf.md:18]:
```suggestion
---
Title: 'math.erf()' 
Description: 'Returns the error function of a number.' 
Subjects: 
  - 'Computer Science'
  - 'Data Science'
  - 'Discrete Math'
  - 'Machine Learning'
Tags: 
  - 'Arithmetic'
  - 'Functions'
  - 'Math'
  - 'Statistics'
CatalogContent: 
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Comment (mamtawardhani) [reply content/python/concepts/math-module/terms/math-erf/math-erf.md:18]: Description should start with a capital letter

File: content/python/concepts/math-module/terms/math-erf/math-erf.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The **`math.erf()`** function returns the error function of a number.
+
+## Syntax
+
+```pseudo
+math.erf(n)
+```
+
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-erf/math-erf.md:27]:
```suggestion

- `n`: A real number for which the error function is to be computed.

```

Comment (mamtawardhani) [reply content/python/concepts/math-module/terms/math-erf/math-erf.md:27]: added parameter and description

File: content/python/concepts/math-module/terms/math-erf/math-erf.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+The `math.erf()` function accepts any real number from negative infinity to positive infinity as input and returns a value between -1 and 1, representing the error function of that number.
+
+## Example
+
+Use `math.erf()` to return the error function of 1:
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-erf/math-erf.md:32]:
```suggestion
In the example below, `math.erf()` returns the error function of *1*:
```

File: content/python/concepts/math-module/terms/math-erf/math-erf.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+```py
+import math
+
+print(math.erf(1))
+```
+
+The above code gives the following output: 
+
+```shell
+0.8427007929497149
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-erf/math-erf.md:43]:
```suggestion
0.842700792949715
```

Comment (mamtawardhani) [reply content/python/concepts/math-module/terms/math-erf/math-erf.md:43]: Please recheck this output once?

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @sicxz, thank you for contributing to the Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few changes, could you please review and modfy them at your earliest convenience? Feel free to discuss any concerns, thank you! 😃

File: content/python/concepts/math-module/terms/math-erf/math-erf.md (hunk 4 above)
Comment (sicxz) [reply content/python/concepts/math-module/terms/math-erf/math-erf.md:45]: I got 0.8427007929497149 again. See screenshot. 

Screenshot 2024-08-19 at 11 12 03 AM

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the changes @sicxz 😄 

The entry looks good for a second round of review! 🚀

**Review by sicxz — COMMENTED** [review]
made updates per pull request review
- capitalized letter
- made bold certain areas that I missed on first round
- checked output of the function to ensure result was correct.

**Review by letsbuildanything — APPROVED** [review]
LGTM! 🚀

**Pull request merged**